topdown/crypto.go

```diff
@@ -22,6 +22,15 @@ import (
 	"github.com/open-policy-agent/opa/util"
 )
 
+const (
+	// BlockTypeCertificate indicates this PEM block contains the signed certificate.
+	// Exported for tests.
+	BlockTypeCertificate = "CERTIFICATE"
```

Comment:
❓ Do these need to be exported?

Reply:
Unfortunately we use the `BlockTypeCertificate` constant (and in an identical sense) in `plugins/rest/rest_test.go`. If we want to un-export it, I'm pretty sure the linter won't complain. (Just let me know)

I've now un-exported `BlockTypeCertificatRequest`.

Reply:
Yeah, let's unexport them both. A little duplication there doesn't hurt 🤷

Reply:
Sounds good! Done.